manifests/init.pp

```diff
@@ -146,7 +147,7 @@
 
   include foreman_proxy_content::pub_dir
 
-  if $qpid_router {
+  if $enable_katello_agent {
```

Comment:
I'm wondering if we should implement an `else` for this where we ensure it's stopped.

Reply:
Will we need to add some additional parameters to puppet-qpid to be able to ensure things are stopped? Given https://github.com/theforeman/puppet-qpid/blob/master/manifests/router/service.pp#L6